modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -285,6 +285,8 @@ CV__DNN_INLINE_NS_BEGIN
 
         virtual Ptr<BackendNode> initVkCom(const std::vector<Ptr<BackendWrapper> > &inputs);
 
+		virtual Ptr<BackendNode> initTengine(InputArrayOfArrays inputs_arr, OutputArrayOfArrays outputs_arr);
```

Comment:
looks like indentation is inconsistent. please, fix it. In the public header files it's especially important to keep them clean and accurate

Reply:
use mixedCaseStyle instead of lower_case_style_with_undescores for identifiers  
--------------------------------------
Do you means to modify "InputArrayOfArrays inputs_arr" -to--> "const std::vector<Ptr<BackendWrapper> > &inputs"  ?

Reply:
done